src/test/java/webdrivertasks/waitings/WebElementWaitingManager.java

```diff
@@ -0,0 +1,40 @@
+package webdrivertasks.waitings;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import webdrivertasks.driver.WebDriverSetUp;
+
+public class WebElementWaitingManager {
+
+    private static final int DEFAULT_WAIT_TIMEOUT_SECONDS = 40;
```

Comment:
To my mind, it makes sense to reduce the default timeout, otherwise, we potentially can get really long execution time. If you need longer waiting somewhere - just place it there 🐇